cmd/argocd-application-controller/main.go

```diff
@@ -81,6 +82,8 @@ func newCommand() *cobra.Command {
 
 			settingsMgr := settings.NewSettingsManager(ctx, kubeClient, namespace)
 			kubectl := &kube.KubectlCmd{}
+			legacyDiffDisabled := os.Getenv("ARGOCD_POPULATE_LEGACY_DIFF") == "false"
```

Comment:
Hm, I think `ARGOCD_POPULATE_LEGACY_DIFF` should be `ARGOCD_ENABKE_LEGACY_DIFF`, but that's just me.

Reply:
I agree. `ARGOCD_ENABLE_LEGACY_DIFF` is much better.